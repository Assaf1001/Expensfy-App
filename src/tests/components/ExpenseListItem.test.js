import React from "react";
import { shallow } from "enzyme";
import expenses from "../fixtures/expenses.js";

import ExpenseListItem from "../../components/ExpenseListItem.jsx";

test("Should render ExpenseListItem with expense", () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[0]} />);
    expect(wrapper).toMatchSnapshot();
});
