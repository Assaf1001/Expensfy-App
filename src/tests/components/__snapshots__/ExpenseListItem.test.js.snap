// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseListItem with expense 1`] = `
<div>
  <Link
    to="edit/1"
  >
    <h3>
      Description: 
      Gum
    </h3>
  </Link>
  <p>
    Price: 
    195
  </p>
  <p>
    Created At: 
    0
  </p>
</div>
`;
